front/src/app/components/forms/convention/ConventionSummarySection.tsx

```diff
@@ -237,6 +238,15 @@ const beneficiarySummary = (
         fields["signatories.beneficiary.birthdate"].label,
         displayDate(convention.signatories.beneficiary.birthdate),
       ],
+      ...(convention.internshipKind === "mini-stage-cci" &&
+      convention.signatories.beneficiary.address
```

Comment:
```suggestion
      convention.signatories.beneficiary.address
```
je n'ai pas trouvé d'autres moyens que ça pour pouvoir utiliser addressDtoToString

Reply:
il y a déjà la condition `convention.internshipKind === "mini-stage-cci"` juste en dessous (ligne 250), je pense que tu peux fusionner ton ajout avec